webservices/filters.py

```diff
@@ -5,18 +5,42 @@
 from webservices.common import models
 
 
+def is_exclude_arg(arg):
+    # Handle string and int excludes
+    return str(arg).startswith('-')
+
+
+def parse_exclude_arg(arg):
+    # Integers will come in as negative and strings will start with -
+    if type(arg) is int:
```

Comment:
You might want to consider using [`isinstance`](https://docs.python.org/3/library/functions.html#isinstance) here instead (here's a [detailed explanation why on Stack Overflow](https://stackoverflow.com/a/1549854)), this way if `WebArgs` ever changes how parameters of an `int` type are handled by subclassing, we'll be covered. :-)

As an example, this line would become:

```python
if isinstance(arg, int):
```